### ISSUE TITLE
Not compatible with rollup 0.60

### ISSUE THREAD

**Issue by probil** [ritz078/rollup-plugin-filesize#20]
Without plugin it works fine. But with I got this log:
```
> rollup -c scripts/rollup-build.js


src/index.js → dist/vue-socket.io-ext.esm.js, dist/vue-socket.io-ext.min.js...
[!] TypeError: this.getData is not a function
TypeError: this.getData is not a function
    at Object.ongenerate (/home/circleci/repo/node_modules/rollup-plugin-filesize/dist/index.js:42:21)
    at /home/circleci/repo/node_modules/rollup/dist/rollup.js:21676:58
    at Array.map (<anonymous>)
    at /home/circleci/repo/node_modules/rollup/dist/rollup.js:21675:34
```

**Comment by gkatsev** [comment]
Seeing the same in videojs/video.js#5232

**Comment by probil** [comment]
Quick solution in source.
```js
import fileSize from "filesize";
import boxen from "boxen";
import colors from "colors";
import deepAssign from "deep-assign";
import gzip from "gzip-size";

function render(opt, size, gzip, bundle) {
	const primaryColor = opt.theme === "dark" ? "green" : "black";
	const secondaryColor = opt.theme === "dark" ? "yellow" : "blue";

	return boxen(
		`${bundle.file
			? colors[primaryColor].bold("Destination: ") +
					colors[secondaryColor](bundle.file) +
					"\n"
			: ""}${colors[primaryColor].bold("Bundle size: ")}${colors[
			secondaryColor
		](size)}${opt.showGzippedSize
			? ", " +
					colors[primaryColor].bold("Gzipped size: ") +
					colors[secondaryColor](gzip)
			: ""}`,
		{ padding: 1 }
	);
}

export default function filesize(options = {}) {
	let defaultOptions = {
		format: {},
		theme: "dark",
		render: render,
		showGzippedSize: true
	};

	let opts = deepAssign({}, defaultOptions, options);
	if (options.render) {
		opts.render = options.render;
	}

	const getData = (bundle, code) => {
		let size = fileSize(Buffer.byteLength(code), opts.format);
		let gzipSize = opts.showGzippedSize
			? fileSize(gzip.sync(code), opts.format)
			: "";
		return opts.render(opts, size, gzipSize, bundle);
	};

	return {
		ongenerate(bundle, { code }) {
			console.log(getData(bundle, code));
		}
	};
}
```

But tests are not passed. They are not modular enough. Test rewrite is needed

**Comment by ritz078** [comment]
try v2.0.0

**Comment by probil** [comment]
Awesome) But in any case test should be more modular. What do you think if I will create PR with improvements?

**Comment by ritz078** [comment]
Of course. I just didn't want to delay this as many are stuck in upgrading. I would love a PR for the improvement. I personally am not happy with the current implementation.

**Comment by probil** [comment]
Close issue because it's fixed in version `2.0.0`
